### PR TITLE
change token metadata for single asset + test + readme

### PR DESCRIPTION
**Pull request by frankhillard**

### CODE REVIEW

File: compiled/test.fa2.single_asset.tz [hunk 1]
```diff
@@ -0,0 +1,261 @@
+{ parameter
```
Comment (ziwot) [inline compiled/test.fa2.single_asset.tz]: `compiled` dir should be removed? (also .gitignore is empty, should probably be removed)

File: test/single_asset.extended.test.mligo [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#import "./single_asset.extended.mligo" "ExtendedAsset"
```
Comment (ziwot) [inline test/single_asset.extended.test.mligo:1]: not sure this is usefull, this file is identical to `single_asset.test.mligo`

File: test/single_asset.extended.mligo [hunk 3]
```diff
@@ -14,11 +14,29 @@ type parameter = [@layout:comb]
     | Balance_of of FA2.balance_of
```
Comment (ziwot) [inline test/single_asset.extended.mligo:14]: not sure this is useful in the context of this lib

File: README.md [hunk 4]
```diff
@@ -7,4 +7,30 @@ Original contracts code:
...
+In order to use the library , one must import the library using a package manager. The ligo compiler uses `esy`.
+
+In a `package.json` file, specify the library in the dependencies
+```
+{ "dependencies": { "@ligo/fa": "^1.0.4", "ligo-extendable-fa2": "^1.0.1" } }
+```
+
+Install the library with ligo CLI (with docker)
+```
+@docker run --platform linux/amd64 --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:next install ligo-extendable-fa2
```
Comment (ziwot) [inline README.md]:
```suggestion
ligo install ligo-extendable-fa2
```
the reader can figure out his prefered way to use ligo
we may also just link to [ligo doc](https://ligolang.org/docs/advanced/package-management#consuming)

File: README.md [hunk 5]
```diff
@@ -7,4 +7,30 @@ Original contracts code:
...
+  Big_map.literal ([
+    (0n, { token_id = 0n; token_info = Map.literal([ ("", 0x03) ]) })
+  ]) 
+```
+
 ## How it works?
+
+In order to use the library , one must import the library using a package manager. The ligo compiler uses `esy`.
+
+In a `package.json` file, specify the library in the dependencies
```
Comment (ziwot) [inline README.md]: no need to do it manually, this is done automatically when running `ligo install`

File: README.md [hunk 6]
```diff
@@ -7,4 +7,30 @@ Original contracts code:
 -   <https://gitlab.com/ligolang/contract-catalogue>
 -   <https://github.com/pewulfman/Tezos-TZIP-implementation>
 
+### Token metadata
```
Comment (ziwot) [inline README.md:10]:
```suggestion
## Token metadata
```
(wrong level)

File: README.md (hunk 4 above)
Comment (frankhillard) [reply README.md]: agreed

File: test/single_asset.extended.mligo (hunk 3 above)
Comment (frankhillard) [reply test/single_asset.extended.mligo:14]: Balance_of belongs to the FA2 standard, so it must be defined as an entrypoint of the contract

File: compiled/test.fa2.single_asset.tz (hunk 1 above)
Comment (frankhillard) [reply compiled/test.fa2.single_asset.tz]: agreed

File: test/single_asset.extended.test.mligo (hunk 2 above)
Comment (frankhillard) [reply test/single_asset.extended.test.mligo:1]: this test show an extension of a FA2 , but the other one is a real simple usage with no real extension

File: README.md (hunk 6 above)
Comment (frankhillard) [reply README.md:10]: It is intended do have  ###

**Pull request merged**